flatpak/dev.neovide.Neovide.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.neovide.Neovide</id>
+  <name>Neovide</name>
+  <summary>Advanced graphical interface for Neovim</summary>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <developer id="dev.neovide">
+    <name>Kaylee Simmons</name>
```

Comment:
Does this have to be an individual person? I think it should probably be Neovide Team or something if possible

Reply:
It does not need to be https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer

I think we can use "Neovide Team", that's seems to be quite common when checking other projects in flathub